include/bluetooth/mesh/gen_dtt_cli.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ *
+ * @file
+ * @defgroup bt_mesh_gen_dtt_cli Generic Default Transition Time Client API
+ * @{
+ * @brief API for the Generic Default Transition Time (DTT) Client.
+ */
+
+#ifndef BT_MESH_GEN_DTT_CLI_H__
+#define BT_MESH_GEN_DTT_CLI_H__
+
+#include <bluetooth/mesh/gen_dtt.h>
+#include <bluetooth/mesh/model_types.h>
+
+struct gen_dtt_cli;
+
+/** @def GEN_DTT_CLI
+ *
+ * @brief Initialization parameters for the @ref gen_dtt_cli.
+ *
+ * @param[in] _status_handler Optional status message handler.
+ * @sa gen_dtt_cli::status_handler
+ */
+#define GEN_DTT_CLI(_status_handler)                                           \
```

Comment:
Is this one internal? Same name as the one below, but that has BT_MESH_MODEL_ prefix

Reply:
They serve two different purposes. This one initializes a `struct gen_dtt_cli`, the one below instantiates a `struct bt_mesh_model` for the composition data (the model list).